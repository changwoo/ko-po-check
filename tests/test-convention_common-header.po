# -*-text-*-
# obsolete or deprecated header fields
msgid ""
msgstr ""
"Project-Id-Version: test 15\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-03-06 00:16+0900\n"
"PO-Revision-Date: 2010-03-06 00:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 0);"

msgid ""
msgstr ""
"Project-Id-Version: test 15\n"
"POT-Creation-Date: 2010-03-06 00:16+0900\n"
"PO-Revision-Date: 2010-03-06 00:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;"

msgid ""
msgstr ""
"Project-Id-Version: test 15\n"
"POT-Creation-Date: 2010-03-06 00:16+0900\n"
"PO-Revision-Date: 2010-03-06 00:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;"
