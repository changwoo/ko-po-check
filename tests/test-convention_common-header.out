-:4: Report-Msgid-Bugs-To: 비어 있습니다. 버그 보고하는 메일 혹은 웹 주소를 쓰십시오
-:4: Language-Team: 기본값을 바꾸지 않았습니다. 해당 번역팀 메일 주소를 쓰십시오
-:4: Content-Type: UTF-8 사용을 추천합니다 (charset=UTF-8)
-:4: Plural-Forms: 한국어는 다음과 같이 씁니다: "nplurals=1; plural=0;"
-:17: Language-Team: 필드가 없습니다. 번역 팀이 따로 없으면 본인 주소를 쓰십시오
-:28: Language-Team: 비어 있습니다. 번역 팀이 따로 없으면 본인 주소를 쓰십시오
